server/routes/temporary-accommodation/index.ts

```diff
@@ -22,10 +21,7 @@ export default function routes(controllers: Controllers, services: Services): Ro
   get(paths.static.useNDelius.pattern, staticController.useNDelius(), { auditEvent: 'VIEW_USE_NDELIUS' })
 
   manageRoutes(controllers, services, router)
-
-  if (!config.flags.applyDisabled) {
```

Comment:
Think remove this flag from `config.ts` entirely at this point, since it's no longer used

Reply:
:recycle: